index.html

```diff
@@ -138,13 +138,28 @@ <h1>About Myself</h1>
                         <li class="lang-item"><img src="../src/assets/images/Ellipse_css.svg" alt="img">CSS</li>
                     </ul>
                 </li>
-                <li class="lang-items-title underline">Frameworks<span>></span></li>
-                
-                <li class="lang-items-title underline">Skills<span>></span></li>
-                
+                <li class="lang-items-title underline">Frameworks<span>></span></li>                
+                <li class="lang-items-title underline">Skills<span>></span></li>                
             </ul>
         </section>
-        
+        <section class="contact-form">
+            <img class="contact-bg" src="../src/assets/images/Contact_me_bg.svg" alt="image">
+            <img class="shape-position" src="../src/assets/images/contact_form_shapes_1.svg" alt="img">
+            <div class="contact">
+                <h2 class="contact-header">Contact me</h2>
+                <div class="contact-text">
+                    If you have an application you are interested in developing, a feature that you need built or a
+                    project that needs coding. I’d love to help with it
+                </div>
+                <form action="https://formspree.io/f/xqkjaeyz" method="POST">
+                    <input class="input-field" placeholder="Your name" value="" type="text" id="name" name="user_name" maxlength="30" required>
+                    <input class="input-field" placeholder="Your email" type="email" id="mail" name="user_email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" required>
+                    <textarea class="input-field" rows="6" placeholder="Write something nice" maxlength="500" id="msg" name="user_message" required></textarea>
+                    <button class="contact-btn" type="submit">Get in touch</button>
+                </form>  
+                <div class="footer-img"><img src="./src/assets/images/footer_image.svg" alt="image"></div>              
+            </div>            
+        </section>
```

Comment:
- [Optional] Please consider using indentation of 2 spaces all through your documents for readability. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).